python/lsst/ts/observatory/control/auxtel/latiss.py

```diff
@@ -157,6 +157,8 @@ async def expose(
             key_value_map = f"groupId: {group_id},imageType: {image_type}"
             if test_type is not None:
                 key_value_map += f",testType: {test_type}"
+            else:
+                key_value_map += f",testType: {image_type}"
```

Comment:
Consider adding a comment why you need to do this. Also comment regarding update of docstring

Reply:
I'll add some comments. It's not optional thanks to the gen3 ingest. Learned this the hard way on ComCam.